File: src/main/java/com/zerozero/review/domain/repository/ReviewRepository.java
package com.zerozero.review.domain.repository;

import com.zerozero.review.domain.model.Review;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface ReviewRepository extends JpaRepository<Review, UUID> {

    Boolean existsByUserIdAndStoreId(UUID userId, UUID storeId);

}

File: src/main/java/com/zerozero/review/domain/repository/ReviewRepository.kt
package com.zerozero.review.domain.repository

import com.zerozero.review.domain.model.Review
import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface ReviewRepository : JpaRepository<Review, UUID> {

    fun existsByUserIdAndStoreId(userId: UUID, storeId: UUID): Boolean
}
